utils/repository.py

```diff
@@ -0,0 +1,13 @@
+def convert_list_to_tuple_string(ids_list):
+    validate_list(ids_list)
+    id_value = (
+        str(tuple(ids_list)).replace(",", "")
+        if len(ids_list) == 1
+        else str(tuple(ids_list))
+    )
+    return id_value
+
+
+def validate_list(ids_list):
```

Comment:
Given that we have this two function without context. We could put the asserts in the function `convert_list_to_tuple_string` and make only one function. As they are used only there. What do you think?

Reply:
I'm not sure, I think in that way the function would be doing more things than it should.